fastlane_bot/config/multicaller.py

```diff
@@ -113,7 +113,7 @@ class MultiCaller(ContextManager):
 
     def __init__(self, contract: web3.contract.Contract,
                  web3: Web3,
-                 block_identifier: Any = 'latest', multicall_address = "0x5BA1e12693Dc8F9c48aAD8770482f4739bEeD696"):
+                 block_identifier: Any = 'latest', multicall_address = "0xcA11bde05977b3631167028862bE2a173976CA11"):
```

Comment:
@barakman please make sure it's the right move to replace the default

Reply:
Verified (the only place where this default value is used is in test 899).
Ideally it should be removed, but since there's a default-value argument which appears before this argument, removing the default value of this argument would require removing the default value of the other argument, or replacing the order in which they are passed to the function.